src/components/ModalInfoFilm/ModalInfoFilm.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import PropTypes from "prop-types";
+import ModalTrailer from "../ModalTrailer/ModalTrailer";
+
+import "./style.scss";
+
+const ModalInfoFilm = ({
+  title,
+  rating,
+  genreFilm,
+  overview,
+  handleCloseInfo,
+  handleShowrailer
+}) => {
+  return (
+    <div className="movie-info">
+      <div className="movie-info__close" onClick={handleCloseInfo}></div>
+      <div className="movie-info__rating">
+        <h3>{title}</h3>
+        <span>{rating}</span>
+      </div>
+      <p className="movie-info__genre">
+        {genreFilm &&
```

Comment:
`genreFilm `always returns `true` since we have `defaultProps as []`

Reply:
changed